src/renderer/components/about/index.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="about-dialog" :class="theme">
+    <el-dialog
+      :visible.sync="showAboutDialog"
+      :show-close="false"
+      :modal="true"
+      custom-class="ag-dialog-table"
+      width="400px"
+    >
+      <img class="logo" :src="markTextLogo" height="128" width="128" />
+      <el-row>
+        <el-col :span="24">
+          <h3 class="text fg-color-dark">{{ name }}</h3>
+        </el-col>
+        <el-col :span="24">
+          <div class="text">{{ versionPrefix }}{{ version }}</div>
+        </el-col>
+        <el-col :span="24">
+          <div class="text">Copyright © 2018 Jocs</div>
+        </el-col>
+      </el-row>
+  </div>
+</template>
+
+<script>
+  import { remote } from 'electron'
+  import { mapState } from 'vuex'
+  import bus from '../../bus'
+
+  export default {
+    data () {
+      return {
+        showAboutDialog: false,
+        name: 'Mark Text',
+        version: remote.app.getVersion(),
```

Comment:
Can you put version in store, maybe other components will use this.

Reply:
How do i access the store? Did you mean `state`?

Reply:
yes, use `mapState` in computed state, You can refer other files in this project.